tools/python/maps_generator/README.md

```diff
@@ -12,22 +14,28 @@ Further we will call them world and coastlines.
 ## Setup
 You must have Python version not lower than 3.6 and complete the following steps:
 
-0. [Build and install generator_tool.](https://github.com/mapsme/omim/blob/master/docs/INSTALL.md#maps-generator)
-1. Change directory:
+0. Checkout to branch of your's app version (see note of #maps_generator section). 
+For example, if you use MAPS.ME 9.2 you should do:
+```sh
+git checkout release-92
+```
+You could search for the version in "About" window in settings menu in MAPS.ME
```

Comment:
The app version can be found in the "About" section in the settings menu of MAPS.ME.

Reply:
fixed